smartmeter.py

```diff
@@ -33,6 +34,13 @@ def load_config():
 if CONFIG["telegram_bot"]["token"]:
     from smartmeter_telegrambot import SmartmeterBot
 
+#Only create model if mqtt is configured
+if CONFIG["mqtt"]:
```

Comment:
Das wird so nicht funktionieren, ansonsten müsste der Benutzer immer alle Einträge der Vorlage löschen.
Besser würde ich es finden wenn es hier einen z.B "is_active" Parameter gibt, den man in der Konfigurationsdatei zwischen True und False setzt.